.github/workflows/build-docker-dev-images.yml

```diff
@@ -6,7 +6,7 @@ on:
   workflow_dispatch:
 
 env:
-  VERSION_NUMBER: ${{ github.run_id }}.${{ github.run_number }}
+  VERSION_NUMBER: 0.${{ github.run_id }}.${{ github.run_number }}
```

Comment:
Irgendwie ist die null unnötig und verwirrend, falls wir für prod semantische versionierung verwenden

Reply:
Das Problem ist, dass wir diese Versionsnummer weiter unten in einem npm version verwenden, damit im Frontend die aktuelle build/versionsnummer steht. 
npm version möchte allerdings ein x.x.x haben und schlägt ansonsten fehl. Deshalb ist die 0. dort hinzugekommen.
Bin aber offen für Vorschläge